ethereum_history_api/oracles/src/script/prepareNoirFixtures.ts

```diff
@@ -61,7 +57,7 @@ for (const hardFork in FIXTURES) {
     const block = await client.getBlock({ blockNumber });
     const stateProof = await client.getProof({
       address,
-      storageKeys: storageKeys || [],
+      storageKeys: storageKeys ?? [],
```

Comment:
I wonder if it is still necceary

Reply:
Yes, as some of our fixture definitions don't have keys